packages/key-readmodel-writer/src/keyConsumerServiceV2.ts

```diff
@@ -16,7 +12,7 @@ export async function handleMessageV2(
   await match(message)
     .with({ type: "ClientKeyAdded" }, async (message) => {
       const client = message.data.client
-        ? toReadModelClient(fromClientV2(message.data.client))
```

Comment:
There is no need to use `toReadModelClient` (the date converter), because the `clients` collection is not involved in this update, and the `keys` collection doesn't need a date converter.